### PR TITLE
Added graph visualization for tweet sentiment analysis results

### PR DESCRIPTION
**Pull request by saurabh28march**
@maze-runnar  @SARTHAK1SINGH  please review!!

### CODE REVIEW

**Comment by saurabh28march** [comment]
@maze-runnar  description of list is in comments in app.py file

**Comment by maze-runnar** [comment]
not working correctly

**Comment by maze-runnar** [comment]
I have updated demo1 and demo2

**Comment by maze-runnar** [comment]
@SARTHAK1SINGH merge it

File: app.py [hunk 1]
```diff
@@ -102,8 +102,8 @@ def __repr__(self):
 path2 = 'https://drive.google.com/uc?export=download&id='+url2.split('/')[-2]
 
 
-df_train = pd.read_csv(path1)
-df_test = pd.read_csv(path2)
+df_train = pd.read_csv("C:\\Users\\Lenovo\\Desktop\\only programmig\\drugsComTrain_raw.csv")
+df_test = pd.read_csv("C:\\Users\\Lenovo\\Desktop\\only programmig\\drugsComTest_raw.csv")
```
Comment (maze-runnar) [inline app.py:106]: change back to path1 and path2 chhotu

File: templates/explore.html [hunk 2]
```diff
@@ -237,6 +240,47 @@
...
+var chart = new CanvasJS.Chart("chartContainer", {
+  animationEnabled: true,
+  zoomEnabled: true,
+  title:{
+    text: "Sentiment analysis of Tweets"
+  },
+  axisX: {
+    title:"Time",
+    minimum: 0,
+    maximum: 10
```
Comment (maze-runnar) [inline templates/explore.html:258]: change maximum to 20

File: app.py (hunk 1 above)
Comment (saurabh28march) [reply app.py:106]: 2 hi line to hai kar lo next pr me

**Pull request merged**